### PR TITLE
Do not use CellWatcher outside of notebook env

### PR DESCRIPTION
**Pull request by bnmajor**

### CODE REVIEW

File: itkwidgets/viewer.py [hunk 1]
```diff
@@ -40,7 +40,8 @@
 
 _viewer_count = 1
 _codecs_registered = False
```
Comment (thewtex) [inline itkwidgets/viewer.py:42]: Can we have something like:

```
_cell_watcher = None
if ENVIRONMENT is not Env.HYPHA:
    _cell_watcher = CellWatcher() # Instantiate the singleton class right away

[...]
# CellWatcher().add_viewer(self.parent) to
_cell_watcher and _cell_watcher.add_viewer(self.parent)
```

**Review by thewtex — CHANGES_REQUESTED** [review]
@bnmajor thanks!

Minor refactor requested inline.

File: itkwidgets/viewer.py (hunk 1 above)
Comment (bnmajor) [reply itkwidgets/viewer.py:42]: Sure thing!

**Review by thewtex — APPROVED** [review]
:100:

**Pull request merged**